astro/src/content/blog/fusionauth-funding.mdx

```diff
@@ -0,0 +1,33 @@
+---
+publish_date: 2023-10-31
+title: Unlocking the Future of CIAM - FusionAuth and Updata Partners Join Forces
+description: The $65 million investment from Updata Partners allows FusionAuth to accelerate its existing growth.
+authors: FusionAuth
+image: /img/blogs/fusionauth-funding/funding-announcement-2023.png
+categories: Company, News
+tags: investment
+excerpt_separator: "{/* more */}"
+---
+We have some exciting news to share. Today, we're announcing the successful completion of a significant $65 million growth equity investment from [Updata Partners](https://updata.com), a prominent growth equity firm.{/* more */} This milestone comes on the heels of widespread developer adoption and marks the first time that we have raised outside capital.
+
+Being a boostrapped company has always been a source of pride for us. However, it can also be a limiting factor. The investment from Updata Partners allows us to move faster than we would have been able to otherwise. As part of this investment round, two folks from Updata, Dan Moss, Principal, and Jon Seeber, General Partner, will join FusionAuth's board, bringing their expertise and insights to the company's journey.
+
+Founded in 2018, FusionAuth has achieved profitability and bootstrapped its way to success prior to this significant investment. The platform has already made its mark across six continents, serving over [450 customers](/customers-partners/?utm_source=blog&utm_medium=blog+post&utm_campaign=funding-announcement), including renowned brands such as Frontdoor, Oppenheimer & Co, Stihl, and Trusted Media Brands. We've grown by over 100% year over year, and we look forward to moving even faster now.
```

Comment:
2018?

Reply:
According to Brian's timeline. That's when we took on the FA name.